### PR TITLE
AirfRANS: lr=1e-4+T_max=10 — ultra-low LR floor test

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

The LR sweep shows a clear trend: 5e-4 → 0.0207, 3e-4 → 0.0197. Does continuing lower help? Test the floor at 1e-4. This LR may converge too slowly to reach the phase transition within the ~41-epoch budget (each cosine cycle at T_max=10 restarts ~750 steps/cycle, so 41 epochs = ~4 full LR cycles). However, if the phase transition does occur, the basin depth could be significantly lower. This is a boundary test: either we confirm the LR floor is too low (no transition → informative failure) or we find a deeper minimum.

## Instructions

Run exactly this command:
```bash
cd target/icml2026 && python train.py --dataset airfrans --airfrans_task full --optimizer adamw --lr 1e-4 --cosine_t_max 10 --no-use-ema --enable-fourier --epochs 999 --wandb_group airfrans-lr-sweep
```

The model will run until the SENPAI timeout (~30 min / ~41 epochs based on prior runs at this config). Report:
1. `val_primary/surface_mse` at best epoch
2. Whether a phase transition occurred (sudden drop in val loss) and if so, at which epoch
3. If no transition: the final epoch val_primary/surface_mse and training trajectory

## Baseline

- **Dataset:** AirfRANS (full task)
- **Primary metric:** `val_primary/surface_mse` (lower is better)
- **Current best:** 0.0197 (PR #2614, gilbert, lr=3e-4, T_max=10, 41 epochs, W&B run: v5ka7832)
- **lr sweep context:** lr=5e-4 → 0.0207, lr=3e-4 → 0.0197. Testing if lr=1e-4 continues the trend.
- **External target:** 0.0043 (~4.6x gap remaining)
- **Reproduce baseline:** `cd target/icml2026 && python train.py --dataset airfrans --airfrans_task full --optimizer adamw --lr 3e-4 --cosine_t_max 10 --no-use-ema --enable-fourier --epochs 999`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2658 because ADVISOR: Closing — lr=1e-4 is a confirmed dead end (too slow). Baseline now 0.01419 via grad-clip=1.0+lr=7e-4. nezuko, please pick up your next assignment..
